security.md

```diff
@@ -37,3 +37,90 @@ We are working to identify and coordinate security efforts across the Jupyter co
 The [Jupyter Security](https://github.com/jupyter/security) GitHub repo has information how to participate and contribute.
 For discussion, please use the special Discourse [security topic](https://discourse.jupyter.org/c/special-topics/security/48) on the Jupyter Discourse server.
 
+----
+
+Below is a copy of the vulnerability handling procedure that can be found on the [security repository](https://github.com/jupyter/security).
+
+# Jupyter Vulnerability Handling Process
+
+This document summarizes and proposes guidelines for handling vulnerabilities reported in Jupyter projects. 
+
+Security issues and vulnerabilities have expectations and processes that are differ from typical open source practices:
+ - Private discussions
+ - Obfuscation
+ - Short timeline
+
+This makes it quite hard to be able to understand, learn, or know what to expect from a security point of view. This document
+will give you a glimpse on what's happening on the inside, and what timeline to expect when you report a security vulnerability. 
+It will also serve as a guideline and task list for Jupyter Contributors and Developers on how to handle security relatied issues.
+
+## Scope
+
+This process applies to *all projects* governed by Jupyter, including JupyterLab, Jupyter Notebook, JupyterHub, and Jupyter Server.
+
+## Reporting Vulnerabilities
+
+If you believe you’ve found a security vulnerability in a Jupyter project, please report it to security@ipython.org. If you prefer 
+to encrypt your security reports, you can use [this PGP public key](https://jupyter.org/assets/ipython_security.asc). Project Jupyter 
+will respond within 3 days to all new reports.
+
+## Coordinated Disclosures
+
+Project Jupyter follows a [coordinated disclosure](https://cheatsheetseries.owasp.org/cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.html#responsible-or-coordinated-disclosure)
+model where the initial report and remediation are handled privately, but the completion description is made public once a patch
+is available. Project Jupyter will disclose known vulnerabilities within 90 days by default, whether a patch is available or not.
+
+## Acknowledgement
+
+Project Jupyter will work to ensure that security researchers, developers, users, or others who identify and report vulnerabilities
+within Project Jupyter software receive acknowledgement for their contribution. 
+
+
+## Vulnerability Triage & Remediation Process
+
+This section describes the process used by Project Jupyter to track, remediate, and disclose reported vulnerabilities. 
+This description is both a reference for the Jupyter Community and a guideline for contributors.
+
+### Roles
+
+This process defines these roles:
+- **Reporter** The individual(s) who report the vulnerability 
+- **Coordinator** A Project Jupyter contributor who facilitates the tracking of the vulnerability through this process
```

Comment:
Given the scope of the project, does it makes sense to require each official Subproject to appoint a Security Coordinator? Right now, it is highly ambiguous who those people would be. Having a named person that the security team can pull in to drive the process of addressing the issue in the Subproject would be super helpful. Ping @fperez @afshin as we are also thinking of other working groups and standing committees having such people appointed for each Subproject.